contracts/StorageFees.cdc

```diff
@@ -0,0 +1,180 @@
+import FlowToken from "./FlowToken.cdc"
+
+pub contract StorageFees {
+
+    // Defines the minimum unit (chunk) size of storage in bytes. Storage can only be bought (or refunded) in a multiple of the minimum storage unit.
+    pub var minimumStorageUnit: UInt64
+
+    // Defines the minimum amount of `storage_capacity` an address can have and also the amount every new account has. `minimumAccountStorage` is a multiple of `minimumStorageUnit`.
+    pub var minimumAccountStorage: UInt64
+
+    // Defines the cost of 1 byte of storage in FLOW tokens.
+    pub var flowPerByte: UFix64
+
+    // Defines the cost of purchasing the initial minimum storage in FLOW tokens.
+    pub var flowPerAccountCreation: UFix64
+
+    // Defines the cost of 1 byte of storage in FLOW tokens.
+    pub var refundingEnabled: Bool
+
+    access(contract) var idCounter: UInt64
+
+    pub resource Administrator {
+        pub fun setRefundingEnabled(enabled: Bool) {
+            StorageFees.refundingEnabled = enabled
+        }
+
+        pub fun setMinimumAccountStorage(minimumAccountStorage: UInt64) {
+            pre{
+                minimumAccountStorage % StorageFees.minimumStorageUnit == UInt64(0): "Minimum account storage must be a multiple of the minimum storage unit"
+            }
+            StorageFees.minimumAccountStorage = minimumAccountStorage
+        }
+
+        pub fun setFlowPerByte(flowPerByte: UFix64) {
+            StorageFees.flowPerByte = flowPerByte
+        }
+
+        pub fun setFlowPerAccountCreation(flowPerAccountCreation: UFix64) {
+            StorageFees.flowPerAccountCreation = flowPerAccountCreation
+        }
+    }
+
+    pub resource interface StorageCapacityCapability{
```

Comment:
Let's not use the word "Capability" on resource objects. A "Capability" is a specific type in Cadence, and creating objects that have that name is sure to create confusion for people. StorageCapacityInterface, perhaps. StorageCapacityAccess, maybe.